projects/igniteui-angular/src/lib/accordion/accordion.component.ts

```diff
@@ -212,7 +215,7 @@ export class IgxAccordionComponent implements AfterContentInit, AfterViewInit, O
         }
 
         this.panels.forEach(panel => {
-            if (panel.collapsed) {
+            if (panel.collapsed && !panel.header.disabled) {
```

Comment:
The check for `collapsed` is a bit redundant, as the panels' expand/collapse methods also check that